spec/ransack/adapters/active_record/base_spec.rb

```diff
@@ -335,6 +335,12 @@ def self.sane_adapter?
             expect { s.result.first }.to_not raise_error
           end
         end
+        
+        describe '#ransack_alias' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.